### ISSUE TITLE
Questions regarding reporting the test performance

### ISSUE THREAD

**Issue by JKDomoguen** [yaoyao-liu/meta-transfer-learning#26]
Hi, I am very grateful for sharing your code here. I hope to just ask a question that is related to your CVPR paper (_Meta-Transfer Learning for Few-Shot Learning_). 

The highest performance on the 1-way miniimagenet with meta-hard tasks is 61.2% as reported in the paper. How was this _exactly_ measured? 

I assume that based on your code, after the meta-training phase you run a validation phase using the validation dataset, then after getting the optimal hyper-parameters you run the testing phase? 

Based on your paper and code the testing phase will consist of 600 randomly sampled tasks from the test dataset. Did you run a couple of test-epochs (where an epoch consists of 600 randomly sampled test tasks) then reported the highest/maximum test accuracy across these test epochs? 

Because I am getting the same result--actually higher--61.5% using this same method.

Thank you for your time.

**Comment by yaoyao-liu** [comment]
Hi @JKDomoguen,

Thanks for your interest in our work.

We randomly sample 600 test episodes and evaluate our method on them. This benchmark we follow other related works, e.g., MAML and ProtoNets. Due to the quality of the pre-trained model and the randomness of episode sampling, the final test score might be different. I suggest you run the experiments several times and use the average score. That number would be more reasonable.  

If you have any further questions, feel free to add more comments.
